app/views/CUIViews/JobRetentionSchemeHelpView.scala.html

```diff
@@ -1,40 +0,0 @@
-@*
- * Copyright 2022 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *@
-
-@import config.AppConfig
-@import templates.MainTemplateCUI
-
-@this(main_template: MainTemplateCUI, appConfig: AppConfig)
-
-@()(implicit request: Request[_], messages: play.api.i18n.Messages)
-
-@main_template(
-    title = messages("cui.job.retention.scheme.title"),
```

Comment:
You need to remove any messages from the message file that are used on this page but not used anywhere else